cylc/flow/scripts/cylc.py

```diff
@@ -203,7 +201,7 @@ def get_version(long=False):
 
     Examples:
       *:running                       # All running workflows
-      workflow//*:running             # All running cycles in workflow
+      workflow//*:running             # All running tasks in workflow
```

Comment:
Still need to say (e..g above under the "Filters" heading) that filters only match in the `n=0` window.

Otherwise there could be an expectation that e.g. `cycle/*:failed` will match final-status failed tasks in a past cycle.

Reply:
Selectors can also be applied to workflows, hence the vagueness.

Will add a note for task selection.